src/main/resources/web/app/qs-form.ts

```diff
@@ -56,11 +56,27 @@ export class QsForm extends LitElement {
     }
   })
   private _formData: any;
+  private _queryData: any;
```

Comment:
Seems to me that `_backendData`/`_browserData` would be a better description of what these variables do, wouldn't it?

Reply:
Probably 😃 

btw the URL format is http://localhost:4000/guides/#categories=data&q=orm

Reply:
What do I know though, maybe `#` makes more sense for client side parameters... I'm clueless.

Reply:
with `#` a page is not forced to be refreshed. If we'd want to go with the `?` then we probably can try
```
window.history.pushState("", "", `?${(new URLSearchParams(this._browserData)).toString()}`)
```
instead, this should also not force a refresh. Let's maybe see if others have any opinions on the URL format. 😃